src/emu/debug/debugcpu.cpp

```diff
@@ -1008,6 +1007,37 @@ u64 debugger_cpu::expression_read_memory(void *param, const char *name, expressi
 			break;
 		}
 
+		case EXPSPACE_OPCODE:
+		{
+			device_t *device = nullptr;
+			device_memory_interface *memory;
+
+			if (name != nullptr)
+				device = expression_get_device(name);
+			if (device == nullptr || !device->interface(memory))
+			{
+				device = get_visible_cpu();
+				memory = &device->memory();
+			}
+			if (memory->has_space(AS_DECRYPTED_OPCODES + (spacenum - EXPSPACE_OPCODE)))
```

Comment:
spacenum can only be EXPSPACE_OPCODE at that point, so that expression should be simplified.  Otherwise it looks buggy, since AS_DECRYPTED_OPCODES is 3, the maximum space index at that point (we have 4 total).